sdk/azcore/runtime/policy_api_version.go

```diff
@@ -24,15 +24,18 @@ const (
 	APIVersionLocationHeader = 1
 )
 
-// NewAPIVersionPolicy constructs an APIVersionPolicy. name is the name of the query parameter or header and
+// apiVersionPolicyOptions contains optional parameters for apiVersionPolicy
+type apiVersionPolicyOptions struct{}
```

Comment:
Personally, I think this is less important now that the constructor isn't exported.  It's fine to leave it too, up to you.

Reply:
Agreed, your call Charles.

Reply:
✂️✂️✂️